Samples/HostedApps/PackageSampleHostedApp/README.md

```diff
@@ -0,0 +1,36 @@
+## Introduction
+In this example, there are two projects – first is PackageSampleHostedApp which is the host app in C# and second is the PackageSampleChild that is the child app. PackageSampleHostedApp is derived from a publically availble UWP sample app called [Package](https://github.com/microsoft/Windows-universal-samples/tree/main/Samples/Package). Package sample app was migrated to the latest .Net6.0, WindowsAppSDK and WinUI3 and a new Scenario 4 called "Hosted App Management" was added to demo new functionality in Hosted App Model such as "PackageCatalog notifications" and "the new FindRelatedPackages API to support querying Packages with HostRuntimes & their children." This new sample is called PackageSampleHostedApp. 
+
+# App package information sample
+
+Shows how to get package info by using the Windows Runtime packaging API ([Windows.ApplicationModel.Package](http://msdn.microsoft.com/library/windows/apps/br224667) and [Windows.ApplicationModel.PackageId](http://msdn.microsoft.com/library/windows/apps/br224668)).
+
+In addition to the already existing features, the sample demos new functionality in Hosted App Model such as "PackageCatalog notifications" and "the new FindRelatedPackages API to support querying Packages with HostRuntimes & their children." 
```

Comment:
don't need quotes around the functionality and add links to the specific apis, eg:

such as PackageCatalog notifications [<insert link to api>]and additions to FindRelatedPackages [<insert link to api>] to support querying packages that are dependent upon host runtimes.